planning/behavior_velocity_planner_common/include/behavior_velocity_planner_common/scene_module_interface.hpp

```diff
@@ -106,8 +108,18 @@ class SceneModuleInterface
   boost::optional<int> first_stop_path_point_index_;
   VelocityFactorInterface velocity_factor_;
 
-  void setSafe(const bool safe) { safe_ = safe; }
+  void setSafe(const bool safe)
+  {
+    safe_ = safe;
+    syncActivation();
+  }
   void setDistance(const double distance) { distance_ = distance; }
+  void syncActivation()
+  {
+    if (!rtc_enabled_) {
+      setActivation(isSafe());
+    }
+  }
```

Comment:
[NITS] super minor suggestion you can ignore
```suggestion
    if (!rtc_enabled_) {
      syncActivation()
    }
  }
  void setDistance(const double distance) { distance_ = distance; }
  void syncActivation() { setActivation(isSafe()); }
```